homeassistant/components/hydrawise/__init__.py

```diff
@@ -24,10 +34,12 @@
 NOTIFICATION_ID = "hydrawise_notification"
 NOTIFICATION_TITLE = "Hydrawise Setup"
 
-DATA_HYDRAWISE = "hydrawise"
+DATA_YAML = "yaml"
```

Comment:
This might make sense - but it should be a separate, preliminary PR.

Reply:
Which part? The constant rename, or the key change?

Reply:
Both, create a preliminary PR dedicated to moving from `hass.data[DATA_HYDRAWISE]` to `hass.data[DOMAIN][DATA_YAML]`